CONTRIBUTING.md

```diff
@@ -44,6 +56,7 @@ please consider doing the following:
 * Make sure the [integration tests]
   (https://github.com/osiam/connector4java-integration-tests) pass.
 * Please squash related commits.
+* Please make sure that your pull request adheres to our coding conventions. 
```

Comment:
I wanted to do that too, Sadly I was unable to generate local anchors and link with Markdown that don't make the plain test ugly. I suspect this to be a shortcoming of markdown. As you know I am a huge fan of [Asciidoctor](http://asciidoctor.org/) anyway. But all joking aside I would be grateful for a hin on how to do that.